bedrock/foundation/templates/foundation/trademarks/list.html

```diff
@@ -9,124 +9,242 @@
+    <li>{{ _('SUMO') }}</li>
+    <li>Sunbird</li>
+    <li>{{ _('Thunderbird') }}</li>
+    <li>View Source</li>
+    <li>XUL</li>
+  </ul>
+
+  <h2>{{ _('Logos') }}</h2>
   <table id="logos">
+    <tr>
+      <td><img width="100" height="100" alt="{{ _('Mozilla logo') }}™" src="{{ static('img/trademarks/mozilla-logo-tm.svg') }}" /></td>
+      <th>{{ _('Mozilla logo') }}</th>
+    </tr>
     <tr>
       <td>
-        {{ high_res_img('trademarks/firefox-logo-r.png', { 'alt': _('Firefox logo'), 'width': '100', 'height': '100' }) }}<span>&reg;</span>
+        {{ high_res_img('trademarks/firefox-logo-r.png', { 'alt': _('Firefox logo'), 'width': '100', 'height': '100' }) }}
       </td>
-      <th>{{ _('Firefox logo') }}</th>
+      <th>{{ _('Firefox') }}</th>
     </tr>
     <tr>
-      <td><img width="100" height="100" alt="{{ _('Firefox OS logo') }}" src="{{ static('img/trademarks/FirefoxOS_logo-wordmark.png') }}" /></td>
-      <th>{{ _('Firefox OS logo') }}</th>
+      <td>
+        <img width="100" height="100" alt="{{ _('Firefox Focus Logo') }}™" src="{{ static('img/trademarks/firefox-focus-logo.png') }}" />
+      </td>
+      <th>{{ _('Firefox Focus') }}</th>
     </tr>
     <tr>
-      <td><img width="100" height="100" alt="{{ _('Thunderbird logo') }}®" src="{{ static('img/trademarks/thunderbird_logo_100.png') }}" /><span>&reg;</span></td>
-      <th>{{ _('Thunderbird logo') }}</th>
+      <td><img width="100" height="100" alt="{{ _('Firefox Rocket Logo') }}™" src="{{ static('img/trademarks/logo-rocket-high-res.png') }}" /></td>
+      <th>{{ _('Firefox Rocket') }}</th>
     </tr>
     <tr>
-        <td><img width="100" height="100" alt="{{ _('Mozilla logo') }}™" src="{{ static('img/trademarks/mozilla-logo-tm.svg') }}" /><span>&trade;</span></td>
-        <th>{{ _('Mozilla logo') }}</th>
-      </tr>
+      <td><img width="100" height="100" alt="{{ _('Bugzilla logo') }}™" src="{{ static('img/trademarks/bugzilla-tm.png') }}" /></td>
+      <th>{{ _('Bugzilla') }}</th>
+    </tr>
     <tr>
-      <td><img width="100" height="100" alt="{{ _('Mozilla dino mascot') }}™" src="{{ static('img/trademarks/dino-head.png') }}" /><span>&trade;</span></td>
-      <th>{{ _('Mozilla dino mascot') }}</th>
+      <td><img width="100" height="100" alt="{{ _('Camino logo') }}®" src="{{ static('img/trademarks/camino-logo-r.png') }}" /></td>
+      <th>{{ _('Camino') }}</th>
     </tr>
     <tr>
-      <td>
-        {{ high_res_img('trademarks/logo-wordmark-webmaker.png', { 'alt': _('Webmaker logo'), 'width': '100', 'height': '100' }) }}<span>&reg;</span>
-      </td>
-      <th>{{ _('Webmaker logo') }}</th>
+      <td><img width="100" height="100" alt="{{ _('Campus Reps logo') }}™" src="{{ static('img/trademarks/campusreps-logo.png') }}" /></td>
+      <th>{{ _('Campus Reps') }}</th>
+    </tr>
+    <tr>
+      <td><img width="100" height="auto" alt="{{ _('Cargo logo') }}™" src="{{ static('img/trademarks/cargo-logo.png') }}" /></td>
```

Comment:
Here and elsewhere, `auto` isn't a valid value for the `height` attribute, it needs to be a number. Browsers will just ignore this so it doesn't actually break anything on the page, but it also doesn't add anything. If the height isn't known you can just omit the `height` attribute and it'll have the same effect.